### PR TITLE
Update entrez-direct recipe to use extended-base

### PR DESCRIPTION
**Pull request by MillironX**
This should resolve issues with SSL certificates via openssl (which is one
of the use cases mentioned in
https://bioconda.github.io/contributor/troubleshooting.html#using-the-extended-image).

Fixes: https://github.com/BioContainers/containers/issues/441

### CODE REVIEW

File: recipes/entrez-direct/meta.yaml [hunk 1]
```diff
@@ -17,6 +17,8 @@ requirements:
   build:
     # The installer script downloads statically compiled binaries, so this is effectively just avoiding linter errors
     - {{ compiler('go') }}
+  run:
+    - wget
```
Comment (bgruening) [inline recipes/entrez-direct/meta.yaml:21]: Why is wget needed?

Comment (MillironX) [reply recipes/entrez-direct/meta.yaml:21]: All of the e-utils depend on either `wget` or `curl`. busybox includes `wget`, but `extended-base` doesn't, so this dependency was hidden previously.

**Comment by MillironX** [comment]
@BiocondaBot please add label

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@BiocondaBot please merge

**Pull request merged**